### PR TITLE
Add VueJs templates example

### PR DESCRIPTION
**Pull request by HendrikGrobler**
Based on other examples, tried to differentiate a bit better between input and events.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -358,6 +358,23 @@ Templates in React are written inside the JavaScript file using the [JSX languag
 </PrimaryButton>
 ```
 
+### VueJs
```
Comment (feimosi) [inline README.md]: Could you write it as `Vue.js` ? I think that's the official name.

File: README.md [hunk 2]
```diff
@@ -358,6 +358,23 @@ Templates in React are written inside the JavaScript file using the [JSX languag
...
+
+Component props can be passed in as strings using a literal syntax, e.g `size="big"` or as actual values using `v-bind`, e.g. `v-bind:disabled="true"`.
```
Comment (feimosi) [inline README.md]: What do you think about writing it like this:

Component props can be passed in as: 
- literal  (using strings) e.g `size="big"` 
- dynamic (using `v-bind` with actual values) e.g. `v-bind:disabled="true"`.

File: README.md [hunk 3]
```diff
@@ -358,6 +358,23 @@ Templates in React are written inside the JavaScript file using the [JSX languag
...
+
+:arrow_right: https://vuejs.org/v2/guide/components.html#Literal-vs-Dynamic
+
+Events can be listened for using `v-on` combined with the event name, and a method name as the value, e.g `v-on:click="saveContent"`.
```
Comment (feimosi) [inline README.md]: I think _listened to_ is the valid form.

**Review by feimosi — CHANGES_REQUESTED** [review]
Really good :+1:

File: README.md (hunk 1 above)
Comment (HendrikGrobler) [reply README.md]: Sure, will update all my outstanding PRs to use Vue.js

File: README.md (hunk 2 above)
Comment (HendrikGrobler) [reply README.md]: I think the bullet points work better, happy to make the change.

File: README.md (hunk 3 above)
Comment (HendrikGrobler) [reply README.md]: Yep, will fix it.

**Comment by feimosi** [comment]
Perfect, then it's ready to be merged :+1:

**Pull request merged**